### PR TITLE
Create where.md entry

### LINKED ISSUE #6142: [Term Entry] PyTorch Tensor Operations: .where()
### Reason/inspiration (optional)

We would like a new entry on the `.where()` term under PyTorch. The entry should go in a new file under `docs/content/pytorch/concepts/tensor-operations/terms/where/where.md`.

The entry should include:

- An introduction to the concept
- A `Syntax` section that provides the syntax for the concept
- An `Example` section that provides an example demonstrating the concept in use

Please refer to the [term entry template](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [x] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by arisdelacruz**
### Description

Attempt to create the .where() entry

### Issue Solved

Closes #6142 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/pytorch/concepts/tensor-operations/terms/where/where.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+Title: '.where()'
+Description: 'Returns a new tensor with elements selected from either `input` or `other`, depending on the condition.'
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensor-operations/terms/where/where.md:3]:
```suggestion
Description: 'Returns values from one tensor if a condition is true; otherwise, it takes values from another tensor.'
```

Comment (mamtawardhani) [reply content/pytorch/concepts/tensor-operations/terms/where/where.md:3]: Since we haven't syntax, having `input` or `other` could be confusing, so reframed it for better readability.

File: content/pytorch/concepts/tensor-operations/terms/where/where.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+Title: '.where()'
+Description: 'Returns a new tensor with elements selected from either `input` or `other`, depending on the condition.'
+Subjects:
+  - 'Computer Science'
+  - 'Machine Learning'
+Tags:
+  - 'Python'
+  - 'Machine Learning'
+  - 'Functions'
+  - 'Tensor'
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensor-operations/terms/where/where.md:11]:
```suggestion
  - 'Functions'
  - 'Machine Learning'
  - 'Python'
  - 'Tensor'
```

Comment (mamtawardhani) [reply content/pytorch/concepts/tensor-operations/terms/where/where.md:11]: arranged in alphabetical order

File: content/pytorch/concepts/tensor-operations/terms/where/where.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+CatalogContent:
+  - 'intro-to-py-torch-and-neural-networks'
+  - 'paths/computer-science'
+  - 'paths/machine-learning'
+---
+
+In PyTorch, the **`.where()`** function returns a new tensor with elements selected from either `input` or `other`, depending on the condition. The function is part of the `torch` module.
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensor-operations/terms/where/where.md:18]:
```suggestion
In PyTorch, the **`.where`** function returns a new [tensor](https://www.codecademy.com/resources/docs/pytorch/tensors) with elements chosen based on a condition, selecting values from one source when the condition is met and from another when it is not.
```

File: content/pytorch/concepts/tensor-operations/terms/where/where.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+## Syntax
+
+```pseudo
+torch.where(condition, x, y) → Tensor
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensor-operations/terms/where/where.md:23]:
```suggestion
torch.where(condition, input, other) → Tensor
```

File: content/pytorch/concepts/tensor-operations/terms/where/where.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+```
+
+- `condition`: The condition tensor.
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensor-operations/terms/where/where.md:26]:
```suggestion
- `condition`: The boolean tensor that controls selection.
```

File: content/pytorch/concepts/tensor-operations/terms/where/where.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+- `x`: The tensor to select elements from when the condition is `True`.
+- `y`: The tensor to select elements from when the condition is `False`.
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensor-operations/terms/where/where.md:28]:
```suggestion
- `input`: The tensor to select elements from when the condition is `True`.
- `other`: The tensor to select elements from when the condition is `False`.
```

File: content/pytorch/concepts/tensor-operations/terms/where/where.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+## Returns
+
+A tensor of elements selected from either `x` or `y`, depending on the condition.
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensor-operations/terms/where/where.md:32]:
```suggestion
It returns a tensor of elements selected from either `input` or `other`, based on the condition.
```

File: content/pytorch/concepts/tensor-operations/terms/where/where.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+The above code produces the following output:
+
+```shell
+tensor([[1, 8],
+        [7, 4]])
+```
+
+In this example, the `.where()` function selects elements from the `x` tensor where the condition tensor is `True` and from the `y` tensor where the condition tensor is `False`. The resulting tensor is created by selecting elements from `x` or `y` based on the condition tensor.
+
+The `.where()` function is particularly useful in conditional operations where you need to select elements from different tensors based on a condition.
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensor-operations/terms/where/where.md:62]:
```suggestion
The `.where()` function is particularly useful in conditional operations where element-wise selection between tensors is required.
```

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @arisdelacruz , thank you for contributing to Codecademy Docs, the entry is nicely written! 😄 

I've suggested a few changes, could you please review and modify those at your earliest convenience? Thank you! 😃

**Comment by arisdelacruz** [comment]
Thanks for the guidance.

**Review by mamtawardhani — APPROVED** [review]
Thank you for contributing to Codecademy Docs @arisdelacruz  😄 
The entry looks good for a second round of review! 🚀

**Review by avdhoottt — APPROVED** [review]
Excellent entry @arisdelacruz, LGTM!!
